### PR TITLE
API contract for qr-code-auth 

### PR DESCRIPTION
**Pull request by shreya-mishra**
Created the API contract of POST api call where we were authenticating the user by user-id and device info.

### CODE REVIEW

File: qr-code-auth/README.md
Comment (heyrandhir) [inline qr-code-auth/README.md:1]: will this route have only POST Request 🤔

File: qr-code-auth/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# QrCodeAuth
+
+## Requests
+
+1. POST 
+
+`/qr-code-auth?user-id=${user-id}&device-info={deviceInfo}`
+
+Description: Authenticates the user by verifying the user ID and device information.
+
```
Comment (heyrandhir) [inline qr-code-auth/README.md:10]: Fields :`Params`, `Query`, `Headers`, `Cookie` are missing 👀

File: qr-code-auth/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+```
+
+```
+{
+    "statusCode": 500,
+    "message": "Internal server error."
+}
+```
+
```
Comment (heyrandhir) [inline qr-code-auth/README.md:46]: would be nice to segregate the  success and error responses separately.

File: qr-code-auth/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+#### Request Body:
+
+```
+{
+  "user_id": "<user-id>",
+  "device_info": "<device-info>"
+}
+```
+
+#### Responses:
```
Comment (heyrandhir) [inline qr-code-auth/README.md:20]: what about 400 Bad Request Error

File: qr-code-auth/README.md
Comment (heyrandhir) [inline qr-code-auth/README.md:1]: can you share the Data model link for this collection

File: qr-code-auth/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    "message": "Internal server error."
+}
+```
+
+
+
+
+
+
+
```
Comment (heyrandhir) [inline qr-code-auth/README.md:52]: These extra lines can be removed from the end.

File: qr-code-auth/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# QrCodeAuth
+
+## Requests
+
+1. POST 
+
+`/qr-code-auth?user-id=${user-id}&device-info={deviceInfo}`
```
Comment (heyrandhir) [inline qr-code-auth/README.md:7]: Nit: A better URL path I feel would be`/qr-code-authentication`. It is more descriptive and accurate than `/qr-code-auth`.

**Review by heyrandhir — CHANGES_REQUESTED** [review]

File: qr-code-auth/README.md (hunk 2 above)
Comment (shreya-mishra) [reply qr-code-auth/README.md]: resolved it in last commit

File: qr-code-auth/README.md (hunk 4 above)
Comment (shreya-mishra) [reply qr-code-auth/README.md]: done

File: qr-code-auth/README.md (hunk 3 above)
Comment (shreya-mishra) [reply qr-code-auth/README.md]: added

File: qr-code-auth/README.md (hunk 1 above)
Comment (shreya-mishra) [reply qr-code-auth/README.md:10]: done

**Review by shreya-mishra — COMMENTED** [review]
resolved conflicts

**Pull request merged**